packages/bloc/lib/src/bloc.dart

```diff
@@ -25,9 +25,9 @@ abstract class Bloc<Event, State> extends Cubit<State>
   /// The current [BlocObserver].
   static BlocObserver observer = BlocObserver();
 
-  final _eventController = StreamController<Event>.broadcast();
+  late final _eventController = StreamController<Event>.broadcast();
```

Comment:
`late` seems a bit redundant here, as we are immediately accessing the variable in our `_bindEventsToStates`
```suggestion
  final _eventController = StreamController<Event>.broadcast();
```